DEVELOPER.md

```diff
@@ -351,18 +351,49 @@ The e2e tests use the [kuttl](https://github.com/kudobuilder/kuttl/) testing fra
    make run-int-tests | tee kuttl.out
    ```
 
-### Run Individual Tests
+### Run Individual e2e Tests
 
 You can run individual tests from the command line with the [KUTTLE CLI](https://kuttl.dev/docs/cli.html#setup-the-kuttl-kubectl-plugin).
 
-> **NOTE**
-> Before you can run an individual test, you need to set up a communal endpoint. To set up a [MinIO](https://min.io/) endpoint, run the following make target:
+Prerequisite steps:
+
+1. Before you can run an individual test, you need to set up a communal endpoint. To set up a [MinIO](https://min.io/) endpoint, run the following make target:
 >
 > ```shell
 > make setup-minio
 > ```
 >
-> To set up a different communal endpoint, see [Custom communal endpoints](#custom-communal-endpoints).
+To set up a different communal endpoint, see [Custom communal endpoints](#custom-communal-endpoints).
+
+2. You also need to set up some environmental variables:
+>
+> ```shell
+> export VERTICA_DEPLOYMENT_METHOD=vclusterops
+> export LICENSE_FILE=/file_path/license.key
+> export BASE_VERTICA_IMG=opentext/vertica-k8s:24.2.0-1
+> export VERTICA_IMG=opentext/vertica-k8s:latest
+> ```
+>
+If the total number of nodes used in a test case is more than 3, you have to set up LICENSE_FILE with the license file path.
+If it is no more than 3, you must unset LICENSE_FILE:
+>
+> ```shell
+> unset LICENSE_FILE
+>
+BASE_VERTICA_IMG and VERTICA_IMG are used for the upgrade test cases. The BASE_VERTICA_IMG is the base vertica vertion that will be installed. VERTICA_IMG is the vertica version that the base version will be upgraded to. The version in VERTICA_IMG must be higher than that in BASE_VERTICA_IMG.
```

Comment:
Could you move the explanation of the environment variables right below the examples mentioned above?
For example:
```shell
export VERTICA_DEPLOYMENT_METHOD=vclusterops
VERTICA_DEPLOYMENT_METHOD=vclusterops lets the backend use ...
```shell
export LICENSE_FILE=/file_path/license.key
If the total number of nodes used ...
...